omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/VisitDetailsFragmentControllerTest.java

```diff
@@ -83,7 +83,7 @@ public void shouldReturnEncountersForVisit() throws ParseException {
       authenticatedUser.addRole(createRoleForUser());
 
       AdministrationService administrationService = mock(AdministrationService.class);
-      when(administrationService.getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATETIME_FORMAT)).thenReturn(
+      when(administrationService.getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATETIME_FORMAT, "dd.MMM.yyyy, HH:mm:ss")).thenReturn(
```

Comment:
@dkayiwa , look at the transition of UiFramework module from this [line vesioin 3.9.0 - 3.11.0](https://github.com/openmrs/openmrs-module-uiframework/blob/3.9/api/src/main/java/org/openmrs/ui/framework/UiFrameworkUtil.java#L625)  to this [line version 3.12.0 - 3.15.0](https://github.com/openmrs/openmrs-module-uiframework/blob/3.12.0/api/src/main/java/org/openmrs/ui/framework/UiFrameworkUtil.java#L676). Since particular failing test depends on this implementation, changing the version number to current, required panel-beating the test code to satisfy the dependency and avoid build failures.